### ISSUE TITLE
[DAT-60] Screen reader - Confirmation messages after submitting ‘renewal all’ in Current checkouts page

### ISSUE THREAD

**Issue by erinesullivan** [mlibrary/my_account#207]
> VoiceOver speaks the status of renewal to match the visual progress indicator (e.g. 3 of 21 items have been renewed...12 of 21 items have been renewed). However, when the renewals are all done, the confirmation message (shown below) does not sound an `aria-polite` alert to speak the text equivalent of the message “26 items were successfully renewed.” That message is only accessible through the right arrow key and appears in the focus order after the close control for the message. 
> 
> Recommend sending an ARIA-polite alert message to confirm how many items were able to be renewed or not at the end of the renewal action/system process.

> (Warning) Good audio alerts during renewal system processing to match visible progress indicator, however no audio alert heard with VoiceOver to indicate a confirmation of the overall success of renewing materials. Suggest using an aria-live="polite” alert to let screen reader users know the status of renewal for their items (e.g. “14 items renewed, 10 items cannot be renewed, see individual items in the table for details.”).
